src/main/java/com/haxwell/apps/quizki/controllers/UserController.java

```diff
@@ -0,0 +1,106 @@
+package com.haxwell.apps.quizki.controllers;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.haxwell.apps.quizki.entities.User;
+import com.haxwell.apps.quizki.exceptions.UserFieldsNotUniqueException;
+import com.haxwell.apps.quizki.repositories.UserRepository;
+
+@CrossOrigin (origins = "http://localhost:4200")
+@RestController
+@RequestMapping ( value = {"/api/users"} )
+public class UserController {
+	
+	private final UserRepository ur;
+	
+	@Autowired
+	UserController(UserRepository ur){
+		this.ur = ur;
+	}
+	
+	@PostMapping
+	public ResponseEntity<User> create(@RequestBody User user){
+		
+		
+		long nameCount = ur.countByName(user.getName());
+		long emailCount = ur.countByEmail(user.getEmail());
+		
+		if (emailCount != 0 || nameCount != 0) {
+			throw new UserFieldsNotUniqueException(emailCount, nameCount);
+		}
+		
+		User savedusr = ur.save(user);
+		
+		return new ResponseEntity<User>(savedusr, HttpStatus.CREATED);
+	}
+	
+	@PostMapping(
+			value = "/isunique",
```

Comment:
camelCase

Reply:
Ok, being changed. I thought that URL's were not case sensitive?? That was the thinking for not using it anyway, that because of that there might be a conflict somehow?